app/exec/extension/default.ts

```diff
@@ -200,7 +220,7 @@ export class ExtensionBase<T> extends TfCommand<ExtensionArguments, T> {
 				_.set(override, "extensionid", extensionId);
 			}
 			let overrideFileContent = Promise.resolve("");
-			if (overridesFile && overridesFile.length > 0) {
+			if (overridesFile && overridesFile.length > 0 && !manifestJs) {
```

Comment:
if an overridesFile is provided why would we silently ignore it?

Reply:
Documentation in the code (interfaces.ts) and the docs (extensions.md) indicate:  _If this is present then manifests, manifestGlobs, json5, override, and overridesFile are ignored._ 

The reason is that manifestJs solves all of the problems that override and overridesFile were trying to address.  It is a superset of their functionality.

Reply:
@dwilsonactual  Actually thinking more about this, there are valid ADO pipeline publishing scenarios where you may want to use both --manifest-js and --override or --overrides-file to update the version with the build version from the pipeline.  In fact, the implementation of --publisher and --extension-id also depends on this functionality internally.  I will make another PR to restore the --override and --overrides-file functionality.